draft-ietf-tls-sni-encryption.md

```diff
@@ -257,24 +260,22 @@ The master secret is verified by verifying a certificate provided by
 the fronting service, but not by the protected service.
 These solutions expose the client to a Man-In-The-Middle attack by
 the fronting service. Even if the client
-has some reasonable trust in this services, the possibility of
+has some reasonable trust in this service, the possibility of
 MITM attack is troubling.
 
 There are other classes of solutions in which the master secret is verified by
 verifying a certificate provided by the protected service. These solutions offer
 more protection against a Man-In-The-Middle attack by the fronting service.
 
-The fronting service could be pressured
-by adversaries. By design, it could be forced to deny access to
+The fronting service could be pressured by adversaries. 
+By design, it could be forced to deny access to
 the protected service, or to divulge which client accessed it. But
 if MITM is possible, the adversaries would also be able to pressure
 the fronting service into intercepting or spoofing the communications between
 client and protected service.
 
-## Fronting Server Spoofing {#frontingspoofing}
```

Comment:
(just checking that joining the sections is intentional)

Reply:
Yep, intentional!